### PR TITLE
BaSD-MS-71: [REFACTOR] Remove availablility in psychologist

### PR DESCRIPTION
**Pull request by PabloMBernal**

### CODE REVIEW

File: src/Components/Admin/Psychologists/index.js [hunk 1]
```diff
@@ -7,7 +7,6 @@ import styles from './psychologists.module.css';
 import ButtonCreate from 'Components/Shared/Buttons/ButtonCreate';
 import ButtonDelete from 'Components/Shared/Buttons/ButtonDelete';
 import ButtonUpdate from 'Components/Shared/Buttons/ButtonUpdate';
-import ButtonAvailability from 'Components/Shared/Buttons/ButtonAvailability';
 import ModalDelete from 'Components/Shared/Modals/ModalDelete';
 import ModalError from 'Components/Shared/Modals/ModalError';
 import ModalAvailability from 'Components/Shared/Modals/ModalAvailability';
```
Comment (nicolasblasco) [inline src/Components/Admin/Psychologists/index.js:12]: @PabloMBernal we should not import this component

**Comment by nicolasblasco** [comment]
@PabloMBernal we should also remove the modal availability component code in psychologist index

**Comment by nicolasblasco** [comment]
@PabloMBernal we should add a description of the PR

**Comment by nicolasblasco** [comment]
@PabloMBernal this PR is ready to be approved! Change from draft to review.

**Review by nicolasblasco — APPROVED** [review]

**Review by ValentinaCollareta — APPROVED** [review]

**Pull request merged**